moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -360,7 +360,7 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
     considered_start_state_.reset();
   }
 
-  robot_state::RobotStatePtr getStartState()
+  robot_state::RobotStatePtr getStartState() const
```

Comment:
returns pointer to internal pointed-to member. See above.

Reply:
I think giving access to internal member variables without modifying them is a const operation.